python/GafferImageUI/ImageViewUI.py

```diff
@@ -649,10 +793,7 @@ def __mouseMove( self, imageGadget, event ) :
 
 	def __buttonPress( self, imageGadget, event ) :
 
-		if event.buttons == event.Buttons.Left and not event.modifiers :
-			self.__createInspectorStartPosition = None
-			return True # accept press so we get dragBegin() for dragging color
```

Comment:
We've lost the ability to left+drag a pixel value straight off of the image. I suspect this is the cause, but haven't confirmed.